docs/notebooks/failure_ego.pct.py

```diff
@@ -33,6 +33,7 @@
 )
 from util.plotting import plot_gp_2d, plot_function_2d, plot_bo_points
 
+
```

Comment:
why victor?!

Reply:
lines are added automatically when I run the notebook, this is indeed annoying

Reply:
maybe we should just go with it

Reply:
or are they going to keep adding them indefinitely?